src/content/docs/ai-monitoring/view-ai-data.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'View AI data in New Relic'
+metaDescription: 'AI monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import aiIntroAiUi from 'images/ai_screenshot-full_intro-ai-ui.webp'
+
+import aiTimeseriesBillboard from 'images/ai_screenshot-crop_timeseries-billboard.webp'
+
+import aiCroppedImageofAIBillboards from 'images/ai_screenshot-crop_billboard.webp'
+
+import aiCroppedImageofAItimeseries from 'images/ai_screenshot-crop_Cropped-image-of-AI-timeseries.webp'
+
+import aiAIEntitiesPage from 'images/ai_screenshot-crop_AI-entities-page.webp'
+
+import aiTopleveAiResponsesSummary from 'images/ai_screenshot-crop_topleve-ai-responses-summary.webp'
+
+import aiResponseTable from 'images/ai_screenshot-crop_response-table.webp'
+
+import aiTraceViewAiResponse from 'images/ai_screenshot-full_trace-view-ai-response.webp'
+
+import aiTraceWaterfallPageSpanDetails from 'images/ai_screenshot-crop_trace-waterfall-page-span-details.webp'
+
+import aiTraceWaterfallPageErrorDetails from 'images/ai_screenshot-crop_trace-waterfall-page-error-details.webp'
+
+import aiAiModelComparisonPage from 'images/ai_screenshot-full_ai-model-comparison-page.webp'
+
+Enabling AI monitoring allows the agent to recognize and capture performance metrics and trace data about your app's AI-layer. You can track token usage, number of completions, and response time from your AI-powered app. When you see an error or an inaccurate response, scope to a trace-level view on a given prompt-response interaction to identify problems in the logic of your AI service. 
+
+<img
+    title="AI responses data"
+    alt="An image that shows the kind of data you get when you enable AI monitoring"
+    src={aiIntroAiUi}
+/>
+
+You can view your data by going to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring**. You can see your data from three different pages:
+
+* **AI responses**: Overview aggregated data from all your AI entities. Track your AI responses, times, and tokens, or see data about individual prompts and responses. 
+* **AI entities**: View a table summary of all entities reporting AI data. See entities with standard APM data like error rate, throughput, and app response time. When you select an entity, you can start exploring the APM **AI responses** page. 
```

Comment:
I'd say change this to "See entities with standard APM data, such as error rate,..." to prevent confusion. Just a nit-pick, feel free to ignore.